backend/app/health.py

```diff
@@ -1,8 +1,26 @@
 from fastapi import APIRouter, status
+from fastapi.responses import JSONResponse
 
 health_router = APIRouter()
 
 
 @health_router.get("/", status_code=status.HTTP_200_OK)
 def health():
     return "UP"
+
+@health_router.get("/nginx-logs", status_code=status.HTTP_200_OK)
+def nginx_logs():
+    with open('/var/log/qa.homeunite.us/nginx-access.log') as f:
```

Comment:
This is a good place to introduce a `HUU_ENVIRONMENT` environment variable in `config.Settings`.

Reply:
Ah good catch, will change this before merging